proto/warrior.proto

```diff
@@ -57,12 +57,62 @@ message WarriorTalents {
 
 message Warrior {
     message Rotation {
+		enum Type {
+			ArmsSlam = 0;
+			ArmsDW = 1;
+			Fury = 2;
+			General = 3;
+		}
+
+		Type type = 1;
+
+		message ArmsSlamRotation {
+
+			bool use_slam_exec = 1;
+			double slam_latency = 2;
+			bool use_ms_exec = 3;
+		}
+		ArmsSlamRotation arms_slam = 2;
+
+		message ArmsDWRotation {
+
+			bool use_ms_exec = 1;
+		}
+		ArmsDWRotation  arms_dw = 3;
+
+		message FuryRotation {
+			enum PrimaryInstant {
+				Bloodthirst = 0;
+				Whirlwind = 1;
+			}
+			PrimaryInstant primary_instant = 1;
+
+			bool use_bt_exec = 2;
+			double rampage_cd_tresh = 10; // only when using rampage
```

Comment:
Might be a misspelling, should that be 'rampage_cd_thresh'?

If so, if you don't mind lets use 'rampage_cd_threshold' instead. Proto messages are the main interface so variables should be named to be immediately obvious to anyone (even non-warriors  ;)   )

Reply:
Renamed 'rampage_cd_tresh' to 'rampage_cd_threshold' in a21327c1.